backend/src/data/migrations/20220807101240_add_user_name.ts

```diff
@@ -0,0 +1,13 @@
+import { Knex } from 'knex';
+
+async function up(knex: Knex): Promise<void> {
+  return knex.schema.table('users', (table) => {
+    table.string('fullName').notNullable();
```

Comment:
```suggestion
enum ColumnName {
  FULL_NAME: 'full_name'
}

    table.string(ColumnName.FULL_NAME).notNullable();
```
Objection will handle map key (from snake to camel) automatically